aten/src/ATen/native/Convolution.cpp

```diff
@@ -36,7 +36,7 @@ struct ConvParams {
   bool use_cpu_depthwise3x3_winograd(const at::Tensor& input, const at::Tensor& weight) const;
   bool use_cudnn(const at::Tensor& input) const;
   bool use_cudnn_depthwise(const at::Tensor& input, const at::Tensor& weight) const;
-  bool use_miopen(const at::Tensor& input) const;
+  bool use_miopen(const at::Tensor& input, const bool& bias_defined) const;
```

Comment:
Don't pass booleans by const reference, it's pointless

Reply:
Agree.